src/lib/server/discord-data.ts

```diff
@@ -0,0 +1,63 @@
+import { REST } from '@discordjs/rest';
+import { Routes } from 'discord-api-types/v10';
+import { DiscordData } from '../schema';
+
+const discordApiToken = process.env.DISCORD_API_TOKEN || '';
+
+const rest = new REST().setToken(discordApiToken);
+
+export interface IDiscordUser {
```

Comment:
This interface is only used internally.
```suggestion
interface IDiscordUser {
```